be/src/exec/pipeline/exchange/exchange_sink_operator.cpp

```diff
@@ -665,6 +667,8 @@ void ExchangeSinkOperator::close(RuntimeState* state) {
 
 Status ExchangeSinkOperator::serialize_chunk(const Chunk* src, ChunkPB* dst, bool* is_first_chunk, int num_receivers) {
     VLOG_ROW << "[ExchangeSinkOperator] serializing " << src->num_rows() << " rows";
+    auto raw_send_bytes = serde::ProtobufChunkSerde::max_serialized_size(*src, nullptr);
+    COUNTER_UPDATE(_send_bytes_counter, raw_send_bytes * num_receivers);
```

Comment:
max_serialized_size is not an accurate value, shoule we use src->bytes_usage or src->memory_usage?

Reply:
not really, max_serialized_size is closer the accurate value.